packages/mobile/src/invite/saga.ts

```diff
@@ -381,11 +384,13 @@ function* addTempAccountToWallet(inviteCode: string) {
 export function* withdrawFundsFromTempAccount(
   tempAccount: string,
   tempAccountBalanceWei: BigNumber,
-  newAccount: string
+  newAccount: string,
+  currency?: CURRENCY_ENUM
 ) {
   Logger.debug(TAG + '@withdrawFundsFromTempAccount', 'Unlocking temporary account')
   const wallet: RpcWallet = yield call(getConnectedWallet)
-  yield call([wallet, wallet.unlockAccount], tempAccount, TEMP_PW, 600)
+  const password: string = yield call(getPasswordSaga, tempAccount, false, true)
+  yield call([wallet, wallet.unlockAccount], tempAccount, password, 600)
```

Comment:
For other authentication we use an unlock duration of 300- https://github.com/celo-org/celo-monorepo/blob/master/packages/mobile/src/geth/consts.ts#L2
Perhaps worth switching to as the tempAccount could have significantly more funds than the usual invite?

Reply:
+1 use UNLOCK_DURATION const